src/rgw/rgw_zone.cc

```diff
@@ -1096,7 +1096,7 @@ int RGWPeriod::update_latest_epoch(epoch_t epoch)
       return r;
     } else if (epoch <= info.epoch) {
       r = -EEXIST; // fail with EEXIST if epoch is not newer
-      ldout(cct, 1) << "found existing latest_epoch " << info.epoch
+      ldout(cct, 4) << "found existing latest_epoch " << info.epoch
```

Comment:
why "4"?
convention in this file is "0" for "error", "10" for "warning" and "20" for "info/debug"